### PR TITLE
feat(icon): Remove material icons font from production build

### PR DESCRIPTION
**Pull request by allencit**

### CODE REVIEW

File: examples/test-site/src/components/Breadcrumbs/MenuBreadcrumbs.token.tsx [hunk 1]
```diff
@@ -39,21 +39,20 @@ import {
...
+  stylable,
 } from '@bodiless/fclasses';
 import { GatsbyLink } from '@bodiless/gatsby-theme-bodiless';
-
 import {
   asBold,
   asEditableLink,
   asLink,
 } from '../Elements.token';
+//@ts-ignore missing csvg extension declaration
```
Comment (Guilherme-Almeida-Zeni) [inline examples/test-site/src/components/Breadcrumbs/MenuBreadcrumbs.token.tsx:50]: Hello @lucas-varela ,

Have you checked the possibility to configure tsconfig to allow this csvg file extension instead of using ts-ignore here?

Comment (Guilherme-Almeida-Zeni) [reply examples/test-site/src/components/Breadcrumbs/MenuBreadcrumbs.token.tsx:50]: Also, in case it's possible, you would avoid ts-ignore for this kind of lint error in all other files where it appears.

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

File: examples/test-site/src/components/Carousel/icons/Pause.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright © 2020 Johnson & Johnson
```
Comment (Guilherme-Almeida-Zeni) [inline examples/test-site/src/components/Carousel/icons/Pause.tsx:2]: Hello @lucas-varela ,

Could you please update year to 2022 to all new files you are including?

File: examples/test-site/src/images/search_black_24dp.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#a0aec0"><path d="M0 0h24v24H0z" fill="none"/><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/></svg>
```
Comment (Guilherme-Almeida-Zeni) [inline examples/test-site/src/images/search_black_24dp.svg:1]: Hello @lucas-varela ,

Is this file still necessary since you have turned icons into component?

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

File: examples/test-site/src/components/Carousel/icons/Pause.tsx (hunk 2 above)
Comment (lucas-varela) [reply examples/test-site/src/components/Carousel/icons/Pause.tsx]: Nice catch, just updated to 2022.

File: examples/test-site/src/images/search_black_24dp.svg (hunk 3 above)
Comment (lucas-varela) [reply examples/test-site/src/images/search_black_24dp.svg]: It wasn't being used, so I deleted it.

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

**Review by Guilherme-Almeida-Zeni — APPROVED** [review]

**Review by dewen — APPROVED** [review]
